js/modules/k6/http/http_request.go

```diff
@@ -109,9 +117,14 @@ func (h *HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.Stat
 		URL:    url.URL,
 		Header: make(http.Header),
 	}
+	HTTPReq := &HTTPRequest{
```

Comment:
Could we change the name to `respReq` or something more descriptive (and start with lowercase letter to mainntain similar var name style)?

Reply:
:+1: